volatility3/framework/constants/linux/__init__.py

```diff
@@ -5,11 +5,15 @@
 
 Linux-specific values that aren't found in debug symbols
 """
+from enum import IntEnum
 
 KERNEL_NAME = "__kernel__"
 
 # arch/x86/include/asm/page_types.h
 PAGE_SHIFT = 12
+PAGE_SIZE = 1 << PAGE_SHIFT
```

Comment:
So... are these values distinct from the various values the layer has (ie, if a layer said its pages were a different size, would linux use these values, or those from its architecture?)  Just trying to figure out, since we do have these values exposed as part of the intel layer (for example https://github.com/volatilityfoundation/volatility3/blob/develop/volatility3/framework/layers/intel.py#L72).  Happy if linux decides to hard code these and we want one place to keep them all (but we then may also want to comb through the other linux plugins and make them use the value if they have a different way of figuring it out)...

Reply:
Yes, `PAGE_SHIFT` (already there) and its derivatives `PAGE_SIZE`  and `PAGE_MASK` are architecture specific. I will move everything to the Intel layer.